app/components/CapitalProjectDetailPanel/CapitalProjectDetailPanel.tsx

```diff
@@ -0,0 +1,181 @@
+import { useState } from "react";
+import { getYear, getMonth, compareAsc } from "date-fns";
+import numbro from "numbro";
+import { ChevronLeftIcon } from "@chakra-ui/icons";
+import {
+  Box,
+  Flex,
+  Heading,
+  HStack,
+  Text,
+  Wrap,
+  WrapItem,
+  IconButton,
+  Hide,
+} from "@nycplanning/streetscape";
+import { CapitalProjectBudgeted, Agency } from "../../gen";
+
+export interface CapitalProjectDetailPanelProps {
+  capitalProject: CapitalProjectBudgeted;
+  agencies: Agency[];
+  onClose: () => void;
+}
+
+export const CapitalProjectDetailPanel = ({
+  capitalProject,
+  agencies,
+  onClose,
+}: CapitalProjectDetailPanelProps) => {
+  const [isExpanded, setIsExpanded] = useState(false);
+  const getFiscalYearForDate = (date: Date): number => {
+    const year = getYear(date);
+    const month = getMonth(date);
+    return month <= 6 ? year : year + 1;
+  };
+
+  const formatFiscalYearRange = (minDate: Date, maxDate: Date) => {
+    if (compareAsc(minDate, maxDate) === 0) {
```

Comment:
This seems like fun opportunity for a ternary operator. But, that's purely a matter of preference